src/app.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable quotes */
+/* eslint-disable curly */
+/* eslint-disable comma-dangle */
+/* eslint-disable operator-linebreak */
+/* eslint-disable new-cap */
+
+import renderTasks from "./todo";
+import * as myDom from "./dom";
+import clearElement from "./utilis";
+
+let lists =
+  JSON.parse(localStorage.getItem(myDom.LOCAL_STORAGE_LIST_KEY)) || [];
+let selectedListId = localStorage.getItem(
+  myDom.LOCAL_STORAGE_SELECTED_LIST_ID_KEY
+);
+const renderList = () => {
+  lists.forEach((list) => {
+    const listElement = document.createElement("li");
+    listElement.dataset.listId = list.id;
+    listElement.classList.add("list-name");
+    listElement.innerText = list.topic;
+    if (list.id === selectedListId) {
+      listElement.classList.add("active-list");
+    }
+    myDom.listsContainer.appendChild(listElement);
+  });
+};
+const renderTaskCount = (selectedList) => {
+  const incompleteTaskCount = selectedList.tasks.filter(
+    (task) => !task.complete
+  ).length;
+  const taskString = incompleteTaskCount === 1 ? "task" : "tasks";
+  myDom.listCountElement.innerText = `${incompleteTaskCount} ${taskString} remaining`;
+};
+const render = () => {
```

Comment:
- [ ] Under my projects section, a default project should exist even after creating new projects. This has not been implemented yet.